pkg/riff/commands/application.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package commands
+
+import "github.com/spf13/cobra"
+import "github.com/projectriff/riff/pkg/riff"
+
+func NewApplicationCommand(p *riff.Params) *cobra.Command {
```

Comment:
having the main subcommands compose their own subcommands is a great idea 👍

Reply:
This is one of the areas where we should definitely borrow from `knctl` IMO

Reply:
Or knative/client, is `knctl` going to be around once the `client` gains more traction?